.tool-versions

```diff
@@ -0,0 +1,2 @@
+elixir 1.9.4-otp-22
+erlang 22.2.2
```

Comment:
We don't want to include this file in our changes because we don't want to impose it on the upstream project; we should only use that locally.

Reply:
Not sure how this was added.